spec/math/statistics/vector_spec.rb

```diff
@@ -383,6 +383,14 @@
       expect(acf[3]).to be_within(0.001) .of(0.486)
     end
   end
+  
+  context "#percent_change" do
+    it "calculates percent change" do
+      vector = Daru::Vector.new([4,6,6,8,10],index: ['a','f','t','i','k'])
```

Comment:
What if there's missing data in the vector? Vectors with all numeric data and missing data still qualify as numeric vectors.